specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1728,6 +2148,17 @@
       "pattern": "^[^*#&+:<>?]+$",
       "x-ms-parameter-location": "method"
     },
+    "ApiIdRevParameter": {
```

Comment:
>ApiIdRevParameter [](start = 5, length = 17)

why split this into a new Parameter?

Reply:
Because some apiIds should not include the revision.  e.g.  /apis/{apiId}/releases and /apis/{apiId}/revisions